Sources/GRPC/ClientConnection.swift

```diff
@@ -536,7 +549,8 @@ extension ClientConnection {
       connectionKeepalive: ClientConnectionKeepalive = ClientConnectionKeepalive(),
       connectionIdleTimeout: TimeAmount = .minutes(5),
       callStartBehavior: CallStartBehavior = .waitsForConnectivity,
-      httpTargetWindowSize: Int = 65535
+      httpTargetWindowSize: Int = 65535,
+      backgroundActivityLogger: Logger = Logger(label: "io.grpc", factory: { _ in SwiftLogNoOpLogHandler() })
```

Comment:
Is it worth encapsulating this in a `static func` on `Logger` we define internally, just so we have one-and-only-one place to write this code?

Reply:
It has to be `public` as it's used in a `public` signature. I didn't think making it underscored-internal was worth it in this case.

Reply:
Ok, fair enough. :smile: